src/network/tc/qdisc.h

```diff
@@ -0,0 +1,35 @@
+/* SPDX-License-Identifier: LGPL-2.1+
+ * Copyright © 2019 VMware, Inc. */
```

Comment:
We mostly got rid of those ugly and pointless copyright markers. Are those necessary here?

Reply:
Yes this is required in terms of vmware upstream contribution.